seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumComJSRunner.scala

```diff
@@ -93,10 +115,12 @@ class SeleniumComJSRunner(browserProvider: SeleniumBrowser,
          |      receiveBuf = null;
          |    },
          |    send: function(msg) {
-         |      sendMsgBufIn.push(msg);
+         |      var encodedMsg =
+         |        msg.split("\7").join("\7\7").split("\0").join("\7");
```

Comment:
Ah well ... I guess there's the regexp-based alternative, then. http://stackoverflow.com/a/17606289/1829647
Reportedly, it's faster.